test-harness/src/main/java/edu/umd/cs/findbugs/AnalysisRunner.java

```diff
@@ -0,0 +1,81 @@
+package edu.umd.cs.findbugs;
```

Comment:
we should probably move this to a more appropriate package... maybe `edu.umd.cs.findbugs.test`?